### PR TITLE
Release: 3.0.0-alpha.1

### PR DESCRIPTION
**Pull request by hanshuebner**

### CODE REVIEW

File: scripts/make-release [hunk 1]
```diff
@@ -1,30 +1,41 @@
 #!/usr/bin/env bash
 
-source "$(dirname "$0")/common.sh"
```
Comment (kikito) [inline scripts/make-release:3]: Perhaps rename this file to `release.sh`?

File: scripts/release-lib.sh [hunk 2]
```diff
@@ -52,8 +54,8 @@ function yesno() {
 
 #-------------------------------------------------------------------------------
 function check_milestone() {
-  if yesno "Visit the milestones page (https://github.com/Kong/kong/milestone) and ensure PRs are merged. Press 'y' to open it or Ctrl-C to quit"; then
-    $browser https://github.com/Kong/kong/milestones
+  if yesno "Visit the milestones page (https://github.com/$GITHUB_ORG/kong/milestone) and ensure PRs are merged. Press 'y' to open it or Ctrl-C to quit"; then
```
Comment (kikito) [inline scripts/release-lib.sh:57]: Probably a lot of these functions can go back to the release.sh file, now that there's only one file and there's no need to share functionality across 3 files any more

File: scripts/make-release (hunk 1 above)
Comment (kikito) [reply scripts/make-release:3]: Jenkins will bark at the filename change; I am almost certain it hardcodes script/make-patch-release somewher

**Review by kikito — APPROVED** [review]
Approved with comments

File: scripts/make-release (hunk 1 above)
Comment (hanshuebner) [reply scripts/make-release:3]: I've updated the Jenkinsfile to fix this.  I've thought about moving everything from `common.sh` to `make-release` but decided against it for now, as make-release neatly contains only the top-level code.

File: scripts/release-lib.sh (hunk 2 above)
Comment (hanshuebner) [reply scripts/release-lib.sh:57]: I agree.  If the split between top-level code in `make-release` and implementation in `release-lib.sh` becomes too bothersome, we can still merge the two into one.

**Pull request closed without merging**